content/reference/rest-api/templates/versions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Templates: versions'
+weight: 10
+
+menu:
+  main:
+    name: 'versions'
+    identifier: 'rest-api-templates-versions'
+    parent: 'rest-api-templates'
+    pre: 'Lists all versions of a template.'
+---
+
+Lists all versions of a template.
+
+# Request
+
+## HTTP Request
+
+```
+GET https://cawemo.com/api/v1/templates/:id/versions
```

Comment:
I would not include this endpoint in the initial iteration of the API. This would only be useful in advanced use cases and could be added later.